packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/bin/server/controllers/users_controller.dart

```diff
@@ -24,26 +31,31 @@ class UsersController extends ApiController {
       _registerHandler,
     );
 
-    // needs to have an access token, mocked for now
+    router.addRequest(
+      RequestType.GET,
+      '/api/users/me',
+      _myUserHandler,
+    );
+
+    router.addRequest(
+      RequestType.POST,
+      '/api/users/me/email/resend-confirmation',
```

Comment:
I think we agreed that a new Registration controller will be introduced and the endpoints will be changed as followed

`/api/register/email/resend-confirmation`

Reply:
Oops... I had made this change locally and somehow missed it when moving code to the brick. Good catch